daemon/c_vol.c

```diff
@@ -40,6 +40,19 @@
 
 #define MOD_NAME "c_vol"
 
+#include <unistd.h>
+#include <string.h>
+#include <sys/mount.h>
+#include <sys/wait.h>
+#include <sys/types.h>
+#include <sys/stat.h>
+#include <sys/syscall.h>
+#include <sys/sysmacros.h>
+#include <fcntl.h>
+#include <errno.h>
+#include <libgen.h>
+#include <linux/dm-ioctl.h>
+
```

Comment:
do not resort the includes, just add dm-ioctl.h

Reply:
Done